### DISCUSSION TITLE
How to use the default Tailwind colours?

### DISCUSSION THREAD

**Discussion by leroyvn** [jpanther/congo#883] [General]
I tried using the [default Tailwind colours](https://tailwindcss.com/docs/customizing-colors) in custom button definitions, *e.g.* by assigning class `bg-amber-400` to an `<a>` tag, but the resulting CSS does not appear in the compiled CSS. Is there a way to make this work?

**Comment by wolfspyre** [comment]
look at the [congo docs on tailwind usage](https://jpanther.github.io/congo/docs/advanced-customisation/#tailwind-configuration) and see if this helps ya… if you’re still stuck, let’s figure out what needs to be clearer in the docs :)

**Reply by leroyvn** [reply]
Hi @wolfspyre, thank you for the pointer. I performed this dev setup and it works well. Actually, for many classes (*e.g.* introducing new colour brightness values). It's when I try to use a colour that is part of the Tailwind default set, and not included in theme definitions, that it fails. At this point, I guess my actual question is rather: when the [docs mention](https://jpanther.github.io/congo/docs/advanced-customisation/#colour-schemes) that Congo uses three colour definitions (`neutral`, `primary` and `secondary`), does this imply that the other definitions are not usable?

**Comment by wolfspyre** [comment]
yep. this is a tailwind config thing. you will have to create yer own tailwind config wrapper 
or at least that’s what i did:
https://wolfspyrelabs.static.wolfspyre.io/hugo-kisso/wplshortcodes/snippits/wplshortcodes/colorgems/

https://gitlab.wolfspyre.io/wolfspyrelabs/hugo-kisso/wplshortcodes
